components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/APIClient.java

```diff
@@ -147,11 +148,12 @@ private ActionInvocationResponse handleResponse(HttpResponse response) {
             case HttpStatus.SC_SERVICE_UNAVAILABLE:
             case HttpStatus.SC_GATEWAY_TIMEOUT:
                 actionInvocationResponseBuilder.errorLog(
-                        "Failed to execute the action request. Received: " + statusCode);
+                        "Failed to execute the action request. Received status code: " + statusCode + ".");
                 actionInvocationResponseBuilder.retry(true);
                 break;
             default:
-                actionInvocationResponseBuilder.errorLog("Unexpected response status code: " + statusCode);
+                actionInvocationResponseBuilder.errorLog("Unexpected response received with status code " + statusCode
```

Comment:
consistently use the colon after status code -> "status code: "

Reply:
Updated in